uniflow/flow/extract/extract_md_flow.py

```diff
@@ -18,7 +18,7 @@ def __init__(self) -> None:
         """Extract md Flow Constructor."""
         super().__init__()
         self._extract_md_op = ExtractTxtOp(name="extract_md_op")
```

Comment:
nit: why `extract_md_flow` has a `ExtractTxtOp`? It looks like `ExtractTxtOp` is a filetype agnostic I/O read. Maybe, we can consider to refactor it to a general `ExtractIoOp` or some other names?